app/assets/stylesheets/db/controllers/edit-requests.scss

```diff
@@ -0,0 +1,14 @@
+body.db-edit_requests-show {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `db-edit_requests-show` should be written in lowercase with hyphens